root/etc/apparmor.d/docker_titus

```diff
@@ -34,6 +34,8 @@ profile docker_titus flags=(attach_disconnected,mediate_deleted) {
   /titus/sshd/usr/sbin/sshd Cx,
 
   profile sshd /titus/sshd/usr/sbin/sshd {
+    # Allow signals from unconfined, usually systemd
+    signal (send,receive) peer="unconfined",
```

Comment:
Can we restrict this to cap sysadmin on the host or root on the host?

Reply:
I don't see how to do that given the grammar:
http://manpages.ubuntu.com/manpages/xenial/man5/apparmor.d.5.html

The only thing it looks like I have access to is peer for signal stuff.

Reply:
should this be signal receive and not send to unconfined?

Reply:
ah right yes